<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunCommandConfig</class>
 <widget class="QWidget" name="RunCommandConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string comment="runCommand conf dialog: command line label">Run Command:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cfg_ApplicationName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string extracomment="runCommand conf dialog: arguments label">Arguments:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStringListEdit" name="cfg_Arguments" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>183</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QStringListEdit</class>
   <extends>QWidget</extends>
   <header>qstringlistedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
